snap/snapcraft.yaml

```diff
@@ -0,0 +1,82 @@
+name: testcraft
+summary: A craft for testing craft-application
+description: |
+  This is a snap for testing craft-application.
+# adopt-info: testcraft
+version: dev
+
+grade: devel
+confinement: classic
+
+base: core24
+platforms:
+  amd64:
+  arm64:
+  armhf:
+  riscv64:
+  s390x:
+  ppc64el:
+
+apps:
+  testcraft:
+    command: bin/python -m testcraft
+
+parts:
+  python:
+    plugin: nil
+    build-attributes:
+      - enable-patchelf
+    stage-packages:
+      - libpython3-stdlib
+      - libpython3.12-stdlib
+      - libpython3.12-minimal
+      - python3.12-minimal
+      - python3-minimal
+  apt:
+    plugin: nil
+    build-attributes:
+      - enable-patchelf
+    stage-packages:
+      - apt
+      - apt-transport-https
+      - apt-utils
+      - binutils
+  python-apt:
+    plugin: nil
+    stage-packages:
+      - python3-apt
+    organize:
+      usr/lib/python3/dist-packages/*: lib/python3.12/site-packages/
+    build-attributes:
+      - enable-patchelf
+  libs:
+    plugin: nil
+    stage-packages:
+      - libgit2-1.7
+    build-attributes:
+      - enable-patchelf
+  testcraft:
+    source: .
+    after: [python-apt, libs]
+    plugin: nil # Real crafts should just use the uv plugin.
+    build-snaps:
+      - astral-uv
+    build-packages:
+      - git
+      - cargo # for building maturin and other rust-based packages
+      - libffi-dev # for building cffi
+      - libgit2-dev # for pygit2
+    build-environment:
+      - UV_NO_BINARY: "1"
```

Comment:
It would make our day-to-day simpler if testcraft could use wheels. Is there a strong reason to build everything from source?

Reply:
There's not. I've made a note that real crafts should have it but otherwise commented it out.